packages/pros/src/lcd/buttons.rs

```diff
@@ -25,27 +32,48 @@ pub fn read_buttons() -> ButtonsState {
     }
 }
 
+#[derive(Debug)]
+/// The three buttons on the llemu
 pub enum Button {
+    /// The left button
     Left,
+    /// The middle button
     Middle,
+    /// The right button
     Right,
 }
 
-pub struct ButtonCallbacks {
+/// The callbacks for the three buttons on the llemu
+struct ButtonCallbacks {
+    /// The callback for when the left button is pressed
     pub left_cb: Option<Box<dyn Fn() + Send>>,
+    /// The callback for when the middle button is pressed
     pub middle_cb: Option<Box<dyn Fn() + Send>>,
+    /// The callback for when the right button is pressed
     pub right_cb: Option<Box<dyn Fn() + Send>>,
 }
 
+impl Debug for ButtonCallbacks {
+    fn fmt(&self, f: &mut core::fmt::Formatter<'_>) -> core::fmt::Result {
+        f.debug_struct("ButtonCallbacks")
+            .field("left_cb", &self.left_cb.is_some())
+            .field("middle_cb", &self.middle_cb.is_some())
+            .field("right_cb", &self.right_cb.is_some())
+            .finish()
+    }
+}
+
 lazy_static::lazy_static! {
-    pub static ref BUTTON_CALLBACKS: Mutex<ButtonCallbacks> = Mutex::new(ButtonCallbacks {
+    // Allow this because the warning is on a type generated by the macro
```

Comment:
This comment was for a now removed `allow` attribute macro. The comment should be removed.